File: java/src/main/java/com/meteroid/models/BillingType.java
// This file is @generated
package com.meteroid.models;

import com.fasterxml.jackson.annotation.JsonValue;
import com.meteroid.Utils.ToQueryParam;

public enum BillingType implements ToQueryParam {
    ADVANCE("ADVANCE"),
    ARREARS("ARREARS");
    private final String value;

    BillingType(String value) {
        this.value = value;
    }

    @JsonValue
    public String getValue() {
        return this.value;
    }

    @Override
    public String toQueryParam() {
        return this.value;
    }
}

File: typescript/src/models/billingType.ts
// this file is @generated

export enum BillingType {
  Advance = "ADVANCE",
  Arrears = "ARREARS",
}

export const BillingTypeSerializer = {
  _fromJsonObject(object: any): BillingType {
    return object;
  },

  _toJsonObject(self: BillingType): any {
    return self;
  },
};
